theme_override/base.html

```diff
@@ -95,8 +95,8 @@
         {% endif %}
       </defs>
     </svg>
-    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="drawer" autocomplete="off">
```

Comment:
This file is a copy of the one provided by our mkdocs theme, hence why it is in the "theme_override" directory. We only made tiny changes to this file which were mostly for #470 and #598. These changes have now been included in the theme by default, so we will actually be removing the file soon, its already deleted in #601 

I therefore suggest we simply revert this change for now.